project-base/app/assets/styles/admin/todo.less

```diff
@@ -95,3 +95,83 @@
 input[type="time"] {
     cursor: text;
 }
+
+.web__header__search {
```

Comment:
those styles should be placed in `packages/framework/assets/styles/admin/todo.less` file